doc/template/simple_form.md

```diff
@@ -46,6 +47,29 @@ Using the [Neat Grid](http://neat.bourbon.io/) to align label and corresponding
     = f.button :submit
 ```
 
+### Field Sizing
+
+As some fields do not require an entire line to themselves (e.g. Postcode), there are 3 different row styles to choose from:
+
+* 2-1
+* 1-2
+* 2-2
+
+Each number refers to the column size input fields can assume in a grid of 7 available columns. The remainder is used for labels. When choosing the sizing, label & input value length should be considered.
+
+**Usage:**
+
+* Extra `.row` and `.sized-[x]-[y]` class for the wrapper of the 2 fields
+
+```haml
+.row.sized-2-1
+  = form.input :suburb
+  = form.input :postcode
+.row.sized-2-2
```

Comment:
I can understand your confusion, however there is not much I can do to provide these form styles otherwise, because I don't know what you want to use them for.

Even if I make an assumption that you will always always only use them for suburb and postcode (2:1), then it will fall short when you have phone numbers with a country prefix because that will be a 1:2 size. Etc. etc etc.

And then, ultimately we will end up with huge selector chains for each case we want to use them in only to make it semantic but underlying it all is the same.

If anyone has a better solution that doesn't end up with shitloads of duplicated and extra code, I'm all for it.